features/bootstrap/FeatureContext.php

```diff
@@ -263,26 +263,26 @@ public function iRunBehatInDebugMode()
      *
      * @Then /^it should (fail|pass) with:$/
      *
-     * @param string       $success "fail" or "pass"
-     * @param PyStringNode $text    PyString text instance
+     * @param 'pass'|'fail' $success
+     * @param PyStringNode  $text    PyString text instance
```

Comment:
I don't think this extra space is needed...

Reply:
I followed the rest of the file (vertical alignment of the types, the variable names and the description)

Reply:
Actually I would remover that entire line, it does not provide any added benefit.

Reply:
@uuf6429 you are right, the information is all redundant